### PR TITLE
Update kubevirtci, add newer cluster-sync provider logic

### PR DESCRIPTION
**Pull request by maya-r**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by akalenyu** [comment]
I was trying to create the 1.21 cluster-sync provider in another PR, there was this little comment about one of the commented lines:
https://github.com/kubevirt/containerized-data-importer/pull/1979#discussion_r725110114

File: tests/framework/pvc.go [hunk 1]
```diff
@@ -111,15 +111,15 @@ func VerifyPVCIsEmpty(f *Framework, pvc *k8sv1.PersistentVolumeClaim, node strin
 	gomega.Expect(err).ToNot(gomega.HaveOccurred())
 	err = f.WaitTimeoutForPodReady(executorPod.Name, utils.PodWaitForTime)
 	gomega.Expect(err).ToNot(gomega.HaveOccurred())
-	output, stderr, err := f.ExecShellInPod(executorPod.Name, f.Namespace.Name, "ls -1 /pvc | wc -l")
+	output, stderr, err := f.ExecShellInPod(executorPod.Name, f.Namespace.Name, "ls -1 /dev/pvc | wc -l")
```
Comment (akalenyu) [inline tests/framework/pvc.go:114]: im not sure we want to change VerifyPVCIsEmpty:
https://github.com/kubevirt/containerized-data-importer/blob/main/tests/upload_test.go#L532

Comment (maya-r) [reply tests/framework/pvc.go:114]: It's a little off

**Pull request merged**